test/asm/ds-byte.asm

```diff
@@ -6,3 +6,4 @@ Label:
 	ds 5, .other - Label - 5 ; Expressions should work...
 	ds 60, .last - Label     ; ...even if not constant
 .last
+	ds 11, $67, $89
```

Comment:
Please add a test for `ds N, @` in both fixed and floating sections. RGBLINK's evaluation of PC may yield surprising results.

Reply:
Ah, I had forgotten about link-time evaluation. You're right, it's different at asm vs link time, even for single `ds 10, @`.

Reply:
\> Mfw we find out a bug we've had for a couple versions

Reply:
Arguably not a bug, or at least, a bug that I don't see an alternative to. It's like how `SIN(someLabel)` is different at link-time than at asm-time if `someLabel` is floating.

Reply:
Could be fixed with a new patch type.

Reply:
Can that be a separate issue+PR from this one?

Reply:
Should be, and I'd prefer to merge it before this one, so that this one can add the relevant tests and pass them.